client/components/section-header/index.jsx

```diff
@@ -34,6 +39,7 @@ export default React.createClass( {
 			<Card compact className={ classes }>
 				<div className="dops-section-header__label">
 					{ this.props.label }
+						<span className="dops-section-header__card-badge">{ this.props.cardBadge }</span>
```

Comment:
@dereksmart Can you write a quick if statement that only adds this span when needed? (otherwise we might get an extra margin)